src/lib/juno.ts

```diff
@@ -12,21 +13,21 @@ class JunoAPI {
 
   get user(): UserAPI {
     if (!this.userAPI) {
-      throw new Error('juno.init() must be called before using the Juno SDK');
+      throw new JunoValidationError('juno.init() must be called before using the Juno SDK');
```

Comment:
Throwing a JunoValidationError instead of a normal Error helps ensure unit tests are catching the correction exception (it could accidentally catch something else and pass otherwise)